frontend/dispatcher.lua

```diff
@@ -66,6 +66,15 @@ local settingsList = {
     toggle_rotation = { category="none", event="SwapRotation", title=_("Toggle orientation"), device=true,},
     invert_rotation = { category="none", event="InvertRotation", title=_("Invert rotation"), device=true,},
     iterate_rotation = { category="none", event="IterateRotation", title=_("Rotate by 90° CW"), device=true, separator=true,},
+    set_refresh_rate = { category="absolutenumber", event="SetBothRefreshRates", min=-1, max=200, title=_("Flash every %1 pages (always)"), device=true,},
+    set_day_refresh_rate = { category="absolutenumber", event="SetDayRefreshRate", min=-1, max=200, title=_("Flash every %1 pages (not in nightmode)"), device=true,},
+    set_night_refresh_rate = { category="absolutenumber", event="SetNightRefreshRate", min=-1, max=200, title=_("Flash every %1 pages (in nightmode)"), device=true,},
+    set_flash_on_chapter_boundaries = { category="none", event="SetToggleFlashOnChapterBoundaries", title=_("Always flash on chapter boundaries"), device=true,},
+    unset_flash_on_chapter_boundaries = { category="none", event="UnsetToggleFlashOnChapterBoundaries", title=_("Never flash on chapter boundaries"), device=true,},
```

Comment:
you might want to combine these into one event that takes on/off true/false, and have one menu item of category="string" for both true and false.

Reply:
Oh, yeah, that's much better, thanks :).

Reply:
Also made me realize I'd typo'd the event name anyway :D.